be/src/exec/hdfs_scanner_text.cpp

```diff
@@ -131,6 +134,16 @@ Status HdfsScannerCSVReader::_fill_buffer() {
     return Status::OK();
 }
 
+void HdfsScannerCSVReader::_trim_row_delimeter(Record* record) {
+    // For default row delemiter which is line break, we need to trim the windows line break
+    // if the file was written in windows platfom.
+    if (_parse_options.row_delimiter == "\n") {
+        while (record->size > 0 && record->data[record->size - 1] == '\r') {
```

Comment:
Why do you remove multiple '\r'? I think only one '\r' should be removed.

Reply:
> Why do you remove multiple '\r'? I think only one '\r' should be removed.

Emm, a character '\r' at the line end is not printable, so I'm not sure is it necessary to keep the extra one because it may cause some strage filtering problem.  I found some other system treat the character '\r' and '\n' as the line end flag and just skip them when parsing the record.